sdk/tables/azure-data-tables/azure/data/tables/_table_client.py

```diff
@@ -369,13 +368,13 @@ def delete_entity(self, *args: Union[EntityType, str], **kwargs: Any) -> None:
 
     @distributed_trace
     def create_entity(self, entity: EntityType, **kwargs) -> Dict[str, Any]:
-        """Insert entity in a table.
+        """Inserts entity in a table.
 
         :param entity: The properties for the table entity.
         :type entity: Union[TableEntity, Mapping[str, Any]]
-        :return: Dictionary mapping operation metadata returned from the service
+        :return: Dictionary mapping operation metadata returned from the service.
         :rtype: dict[str, Any]
-        :raises: :class:`~azure.core.exceptions.HttpResponseError`
+        :raises: :class:`~azure.core.exceptions.HttpResponseError` If the entity already exists
```

Comment:
I think the formatting here should be something like:
```suggestion
        :raises ~azure.core.exceptions.HttpResponseError: If the entity already exists
```
However this doesn't look accurate - shouldn't it be `ResourceExistsError` if the entity already exists and `HttpResponseError` for anything else?

Reply:
* Other places are using `:class:`, I'll keep using it here.
* It could raise other errors than `ResourceExistsError`, so remove "If the entity already exists" should be good.